spyder/utils/icon_manager.py

```diff
@@ -114,7 +112,6 @@
     'bold':                    [('fa.bold',), {'color': MAIN_FG_COLOR}],
     'italic':                  [('fa.italic',), {'color': MAIN_FG_COLOR}],
     'genprefs':                [('fa.cogs',), {'color': MAIN_FG_COLOR}],
-    'exit':                    [('fa.power-off',), {'color': MAIN_FG_COLOR}],
```

Comment:
This icon is still in use

https://github.com/spyder-ide/spyder/blob/8ca177c7eb9bba6c0e1978c221fa4a65a2ea0b8c/spyder/app/mainwindow.py#L927-L931

Please carefully review if the other icons you're trying to remove in this PR are still in use or not.

Reply:
It is redefined on line 205 (formerly 210).

I was maybe not that clear what I actually did. Everything I removed is because of duplicate keys. I have not check if there are actually unused icons here, only duplicate definitions. (I have kept the one with a higher line number as that should be the one kept when running the original code, unless they were identical definitions, then I kept the one which seems to be "best positioned in the code".)

Reply:
Ok, sorry for that. Duplicates probably start to appear thanks to merges or rebases.